src/vs/workbench/contrib/comments/browser/commentNode.ts

```diff
@@ -116,7 +117,7 @@ export class CommentNode<T extends IRange | ICellRange> extends Disposable {
 		const avatar = dom.append(this._domNode, dom.$('div.avatar-container'));
 		if (comment.userIconPath) {
 			const img = <HTMLImageElement>dom.append(avatar, dom.$('img.avatar'));
-			img.src = comment.userIconPath.toString();
+			img.src = FileAccess.uriToBrowserUri(URI.from(comment.userIconPath)).toString(true);
```

Comment:
I'm not sure you need the `FileAccess.uriToBrowserUri`. I'll need to look into it.

Reply:
Definitely need that, because file:// URIs in the DOM are blocked. The line after that one sets an error handler which simply removes the img element when that happens, but if you comment that out you'll see the broken image.